### ISSUE TITLE
Accessibility: Product detail page gallery violates some accessibility rules

### ISSUE THREAD

**Issue by msiewierska** [shopware/shopware#14000]
### Shopware Version

6.7.3.1

### Affected area / extension

Platform(Default)

### Actual behaviour

1.3.1 (level A): all gallery’s elements are read at once (when focus is on group, it should read something shorter, e.g. 'Gallery, 7 items')

2.4.6 (level AA): there are English labels for gallery elements ("slide", "carousel page" etc.)

<img width="805" height="345" alt="Image" src="https://github.com/user-attachments/assets/d804b066-144b-441b-b793-8e62fcafdaf4" />

2.4.6 (level AA) thumbnail carousel: aria-label is not allowed on the generic role

<img width="364" height="535" alt="Image" src="https://github.com/user-attachments/assets/70882492-624f-430c-8e63-a6659cdffa20" />

2.4.6 (level AA)product gallery: images are in taborder, but without specific role

<img width="2150" height="318" alt="Image" src="https://github.com/user-attachments/assets/478c0ae7-61ee-496c-8b3a-6c187231c8e5" />

### Expected behaviour

Product gallery is free of accessibility issues

### How to reproduce

Inspect product gallery on a product with multiple images. Run accessibility audit.

**Comment by nguyenquocdaile** [comment]
Hi @msiewierska, I ran an accessibility audit in Chrome for the PDP with a carousel image, but I found no errors. We have some commits that fixed issues related to the slider image [b517b559c](https://github.com/shopware/shopware/commit/b517b559cffb2094a7b08de759ce75c119677779#diff-f154fbb12bb5d960a85bf679f1ad44d01e4273f2046ee6d9258bdbbdb61468e8) and https://github.com/shopware/shopware/commit/8e43cc0bf37a18e6605424ebabdefcadad6fbfcd. Could you check it again and let me know if the issue still occurs?

<img width="1500" height="818" alt="Image" src="https://github.com/user-attachments/assets/f395791e-2d84-45f7-a741-e2585d803956" />

**Comment by vienthuong** [comment]
Feel free to reopen if the issue still persist with more information
